cassava_trait.obo

```diff
@@ -7594,6 +7594,139 @@ def: "Aroma like tha of sweet potato, Possibly like aroma of boiled, roasted or
 synonym: "BoiCasAromSweetpotat" EXACT []
 is_a: CO_334:0103020 ! Taste trait
 
+[Term]
+id: CO_334:0003040
+name: Fufu colour scale 1-8
+namespace: CassavaTrait
+def: "Measures the intensity of the different colours of the cooked fufu surface on a scale 1 to 8" []
+is_a: CO_334:0001000 ! Variables
+relationship: variable_of CO_334:0003047 ! fufu surface colour
+relationship: variable_of CO_334:0013005 ! appearance method
+relationship: variable_of CO_334:0103022 ! Colour scale 1-8
+created_by: Ugo Chijioke
+
+[Term]
+id: CO_334:0003041
+name: Fufu smoothness scale 0-10
```

Comment:
there are 3 smoothness 1-10 scales. Need to make each one have a unique name or consolidate if the same 

https://github.com/Planteome/CO_334-cassava-traits/blob/3b05da7d5a44d0b02f8f74702c66329f5f4c78c4/cassava_trait.obo#L12155

https://github.com/Planteome/CO_334-cassava-traits/blob/3b05da7d5a44d0b02f8f74702c66329f5f4c78c4/cassava_trait.obo#L12020